go.mod

```diff
@@ -54,9 +54,8 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/ovrclk/cosmos-sdk v0.44.1-patches
```

Comment:
Does v0.44.3 of cosmos SDK incorporate all of our patches?

Reply:
Yeah! We had fixed home flag parsing in our patch branch, which is part of `cosmos v0.44.3` as well.